django/contrib/admin/static/admin/css/base.css

```
@@ -626,22 +630,53 @@ ul.messagelist li {
     display: block;
     font-weight: 400;
     font-size: 0.8125rem;
-    padding: 10px 10px 10px 65px;
-    margin: 0 0 10px 0;
-    background: var(--message-success-bg) url(../img/icon-yes.svg) 40px 12px no-repeat;
-    background-size: 16px auto;
+    padding: 0.625rem 0.625rem 0.625rem 4.063rem;
+    margin: 0 0 0.625rem 0;
+    background-color: var(--message-success-bg);
     color: var(--body-fg);
     word-break: break-word;
+    position: relative;
+}
+
+ul.messagelist li::before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 0.625rem;
+    transform: translateY(-50%);
+    width: 1rem;
+    height: 1rem;
+    mask-image: url(../img/icon-yes.svg);
+    -webkit-mask-image: url(../img/icon-yes.svg);
+    mask-size: contain;
+    -webkit-mask-size: contain;
+    mask-repeat: no-repeat;
+    -webkit-mask-repeat: no-repeat;
```

Do we need both sets of properties? It seems to me setting the size could be enough?

```suggestion
    mask-size: contain;
    -webkit-mask-size: contain;
```

---

No, we don't. Thanks for pointing it out.